gef-ui/server/controllers/about-exporter/index.js

```diff
@@ -1,6 +1,15 @@
 import * as api from '../../services/api';
 import { validationErrorHandler, isTrueSet } from '../../utils/helpers';
 
+const mappedIndustries = (industries, selectedIndustry) => {
+  if (!industries) { return null; }
+  return industries.map((industry) => ({
+    value: JSON.stringify(industry),
+    html: `${industry.name}<br />${industry.class.name}`,
+    checked: selectedIndustry ? JSON.stringify(industry) === selectedIndustry : null,
+  }));
```

Comment:
i'd have any html in the template itself, not a blocker to merge

Reply:
The radio buttons are dynamic, so wont be able to hard code them in. Is there a way to use a for loop for items in the njk file?

Reply:
```
{% for industry in industries %}
  <p>{{ industry.name }}</p>
{% endfor %}
```

Reply:
so this is my template code for the radio buttons:

`{{ govukRadios({
     idPrefix: "industry",
     name: "selectedIndustry",
     items: industries, // Mapped in the controller
     attributes: {
       'data-cy': 'industries'
     }
}) }}`

So are you saying to wrap this macro into a for loop?

Reply:
Sorry, not familiar with design /  didn't realise there were radios involved. I think what you have is ok - I don't think it's possible to populate the govukRadios items attribute dynamically, I think, can only pass in an array and that's it 👍